XRTK-Core/Packages/com.xrtk.core/Editor/PackageInstaller.cs

```diff
@@ -139,20 +139,33 @@ private static void AddConfigurations(List<string> profiles)
 
                 if (platformConfigurationProfile.IsNull()) { continue; }
 
-                var rootProfile = MixedRealityToolkit.IsInitialized
-                    ? MixedRealityToolkit.Instance.ActiveProfile
-                    : ScriptableObjectExtensions.GetAllInstances<MixedRealityToolkitRootProfile>()[0];
-
-                if (EditorUtility.DisplayDialog("We found a new Platform Configuration",
-                    $"We found the {platformConfigurationProfile.name.ToProperCase()}. Would you like to add this platform configuration to your {rootProfile.name}?",
-                    "Yes, Absolutely!",
-                    "later"))
+                MixedRealityToolkitRootProfile rootProfile;
+                if (MixedRealityToolkit.IsInitialized)
                 {
-                    InstallConfiguration(platformConfigurationProfile, rootProfile);
+                    rootProfile = MixedRealityToolkit.Instance.ActiveProfile;
                 }
                 else
                 {
-                    EditorUtility.DisplayDialog("Attention!", "Each data provider will need to be manually registered in each service configuration.", "OK");
+                    var availableRootProfiles = ScriptableObjectExtensions.GetAllInstances<MixedRealityToolkitRootProfile>();
```

Comment:
Do we also need to make this change in all the other package installers?

Reply:
I don't think so. The other platform package installers are using this one in core. The code is shared.